src/backend/distributed/executor/multi_server_executor.c

```diff
@@ -212,7 +212,7 @@ InitTaskExecution(Task *task, TaskExecStatus initialTaskExecStatus)
 	taskExecution->jobId = task->jobId;
 	taskExecution->taskId = task->taskId;
 	taskExecution->nodeCount = nodeCount;
-	taskExecution->connectPollCount = 0;
+	taskExecution->connectStartTime = 0;
```

Comment:
Since `TimestampTz` can be either an integer or double, can you add a cast to `TimestampTz` on the right hand side of this assignment?